app/javascript/src/hyacinth_v1/components/digital_objects/common/DisplayField.js

```diff
@@ -4,6 +4,8 @@ import InputGroup from '../../shared/forms/InputGroup';
 import Label from '../../shared/forms/Label';
 import PlainText from '../../shared/forms/inputs/PlainText';
 
+const randomId = (prefix) => `${prefix}${Math.random()}`.replace('.','');
```

Comment:
Do you mind changing this to the counter method instead?

Reply:
Nope, sounds great.

Reply:
Thanks!